src/ui/menu.ts

```diff
@@ -117,6 +118,23 @@ async function menu(_ui: UI) {
               return;
             });
         return;
+      },
+      popOut: () => {
+        chrome.windows.create({
+          url: chrome.extension.getURL('view/popup.html'),
+          type: 'detached_panel',
```

Comment:
Chrome doesn't support this type for extension:

`Uncaught Error: Invalid value for argument 1. Property 'type': Value must be one of: [normal, popup, panel].`

We need disable this feature on Chrome or change it to another type.